tests/units/cli/test_examples/helloapache/artifacts/marathon/helloapache.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "helloapache",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "network": "BRIDGE",
+      "image": "$image",
+      "portMappings": [
+        {
+          "containerPort": 80,
+          "servicePort": $hostport,
```

Comment:
it is invalid json only before templating

Marathon expects `servicePort` to be integer, if you do `"$hostport"` then after templating it is string and that is invalid for Marathon